nomination/url_handler.py

```diff
@@ -113,112 +117,124 @@ def validate_date(dateinput):
                 continue
         return validdate
 
+
 def add_url(project, form_data):
     summary_list = []
     form_data['url_value'] = check_url(form_data['url_value'])
-    #Get the system nominator
+    # Get the system nominator
     try:
         system_nominator = Nominator.objects.get(id=settings.SYSTEM_NOMINATOR_ID)
-    except:
+    except Nominator.DoesNotExist, Nominator.MultipleObjectsReturned:
```

Comment:
I don't think you could get `Nominator.MultipleObjectsReturned` here.

Reply:
oh right, primary key

Reply:
Yeah I'll remove that from here and the other places you pointed out.